### PR TITLE
Update Community Group Charter for 2024->

### PR DESCRIPTION
**Pull request by anssiko**
- Refresh Goals
- Add Task-specific APIs and Prompt API to Deliverables
- Note WebNN has graduated
- Stylistic tweaks

Note: This is a draft for discussion and review.

[Preview](https://raw.githack.com/webmachinelearning/charter/refs/heads/charter-2024/index.html)

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -113,6 +140,35 @@ <h3 id="neural-network-inference">
...
+    <ul>
+      <li>Classification, tagging, and keyword extraction of arbitrary text;</li>
+      <li>Helping users compose text, such as blog posts, reviews, or biographies;</li>
+      <li>Summarizing, e.g. of articles, user reviews, or chat logs;</li>
+      <li>Generating titles or headlines from article contents;</li>
+      <li>Answering questions based on the unstructured contents of a web page;</li>
+      <li>Translation between languages;</li>
+      <li>Proofreading.</li>
+    </ul>
     <p>
```
Comment (domenic) [inline index.html:172]: Nit: these paragraphs are now under "prompt API". Maybe they should move up to the top?

File: index.html [hunk 2]
```diff
@@ -158,7 +214,25 @@ <h3 id="specifications">
         Web Neural Network API
       </dt>
       <dd>
-        A dedicated API for neural network inference hardware acceleration
+        A dedicated low-level API for neural network inference hardware acceleration.
+      </dd>
+      <dt>
+        Translator API
```
Comment (domenic) [inline index.html:220]: Maybe include "Translator and Language Detector" APIs in this bullet?

**Review by domenic — COMMENTED** [review]
This looks great! Only minor comments from me.

**Comment by anssiko** [comment]
@domenic thanks for the suggestions, PR updated accordingly.

File: index.html [hunk 3]
```diff
@@ -113,16 +150,35 @@ <h3 id="neural-network-inference">
...
+      <li>Rephrase input text.</li>
+    </ul>
+    <h3 id="prompt-api">
+      Prompt API
+    </h3>
     <p>
-      Each specification should detail all known security and privacy
-      implications for implementers, Web authors, and end users.
+      A general-purpose API to prompt a language model directly using common
+      promping techniques to accomplish a variety of tasks, including:
```
[the remark this answers is not in the document]
Comment (HelloFillip) [reply index.html]: A note on this: is privacy and security being reduced / removed as a priority. As someone focused primarily on the privacy implications I would hope to ensure there’s still an emphasis on safety-by-design throughout the process.

**Comment by anssiko** [comment]
Thank you for your review and support everyone! 🚀 

I will merge this PR now with a note this new charter will be operational on 9 December 2024 per the charter amendment process.

**Review by manc95 — COMMENTED** [review]
1

**Review by rashworld-max — APPROVED** [review]

**Pull request merged**